tests/notification/pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
```

Comment:
We should be trying to use the central[ pyproject.toml](https://github.com/mozilla-services/autopush-rs/blob/master/tests/pyproject.toml) instead of having an independent one.

Reply:
This is mainly because the tests are run inside of docker. I could use that file, ill have to make a group just for these tests.